src/NuGet.Services.EndToEnd/NuGet.Services.EndToEnd.csproj

```diff
@@ -101,6 +101,9 @@
     <Content Include="Support\NuGetExe\nuget.4.6.1.exe">
```

Comment:
Is `nuget.4.6.1.exe` still used elsewhere? Does it need to be?

Reply:
Good question. The pattern has been to keep the nuget client around, but I'm not sure why. I'll do some digging.

Reply:
I did a scan through, I couldn't find anything using the previous clients. I went ahead and remove all unnecessary clients.